packages/server/src/service/wisconsinFipsCode.ts

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios'
+import { cache } from './util'
+
+interface Response {
+  features?: {
+    attributes?: {
+      DOA?: string
+    }
+  }[]
+}
+
+const rawWisconsinResponse = async (latLong: [number, number]): Promise<Response | null> => {
+  const [lat, long] = latLong
+  const url = `https://mapservices.legis.wisconsin.gov/arcgis/rest/services/Statewide/Municipal_Boundaries_July_2020/FeatureServer/0/query?where=1%3D1&outFields=*&geometry=${long}%2C${lat}&geometryType=esriGeometryPoint&inSR=4326&spatialRel=esriSpatialRelIntersects&outSR=4326&f=json`
```

Comment:
We're using a open ArcGIS server for this query, I didn't added one of my own (following the instructions from `README.md` since I don't know if the one used at `michiganFipsCode.ts` was created by Marco or using the `production` credentials).

At least here in Brazil this server takes some time to complete responses, so I faced some timeouts on the client side, if we increase the timeout at `env.js` the problem is solved, but I believe having our own private server might decrease the amount of time needed for each request.

Reply:
In the end, that's why we chose to use the private server.  If it's not too much trouble, let's create it.  @Luca409 has the credentials for logging into our ARGGis system and I think it's not too hard to add things into it.